src/content/structured/patterns/side-navigation-layout.mdx

```diff
@@ -557,19 +557,19 @@ With options for:
 
 When choosing a layout pattern, make sure it works for all pages within your app and avoid switching layout patterns on different pages.
 
-Use the side navigation layout to help provide more screen space for main content. Side navigation is well suited to apps that have mutually exclusive features which need to be switched between easily from any page. For example, email providers often use side navigation to provide maximum screen space to email content whilst allowing constant access to other pages.
+Use the side navigation layout to help provide more screen space for main content. Side navigation is well suited to apps that have mutually exclusive features that need to be switched between easily from any page. For example, email providers often use side navigation to provide maximum screen space to email content whilst allowing constant access to other pages.
```

Comment:
```suggestion
Use the side navigation layout to help provide more screen space for main content. Side navigation is well suited to apps that have mutually exclusive features, that need to be switched between easily from any page. For example, email providers often use side navigation to provide maximum screen space to email content whilst allowing constant access to other pages.
```

Reply:
My only concern is that it is a very long sentence. Wonder what the rest of the devs think? 🤔

Reply:
We can't add a comma there as it wouldn't be correct grammar, but if you think it's too long can see a way to split it? I've been having a think but I can't see a way.

Reply:
No problem. Maybe something like this? 

Use the side navigation layout to help provide more screen space for main content. Side navigation is well suited to apps that have mutually exclusive features that need to be switched between easily from any page. For example, email providers often use side navigation to provide maximum screen space to email content. This allows constant access to other pages.

Reply:
Oh sorry, I thought you meant the previous sentence. I see what you mean about this sentence being quite long, but I think removing the whilst changes the meaning - it no longer makes it clear that having the side navigation means there is maximum screen space for the email content 'at the same time as' allowing access to other pages.

Reply:
Fair enough! If anyone else has any ideas, they are welcome 😊 

I would say having a comma before whilst would be good then?

```suggestion
Use the side navigation layout to help provide more screen space for main content. Side navigation is well suited to apps that have mutually exclusive features that need to be switched between easily from any page. For example, email providers often use side navigation to provide maximum screen space to email content, whilst allowing constant access to other pages.
```

Reply:
```suggestion
Use the side navigation layout to help provide more screen space for the main content. Side navigation is well suited to apps that have mutually exclusive features that need to be switched between easily from any page. For example, email providers often use side navigation to provide maximum screen space for email content, allowing constant access to other pages.
```
Another suggestion from me!

Reply:
Thanks. I think removing the whilst again loses the meaning. Would you be happy to keep it in as removing it only makes the sentence shorter by one word? OK to add 'the' and change 'to' to 'for' as suggested.